cclib/parser/molcasparser.py

```diff
@@ -267,19 +267,19 @@ def extract(self, inputfile, line):
         #     5    -37.33596239    -49.47021484     12.13425245 -0.20E-01*  0.59E-01*  0.59E-01*   0.37E+00   0.16E+01   Damp      0.
         if line[46:91] == 'iterations: Energy and convergence statistics':
 
-            self.skip_line(inputfile,'blank')
+            self.skip_line(inputfile, 'blank')
 
-            while line.split() != ['Energy', 'Energy', 'Energy', 'Change', 'Delta', 'Norm', 'in', 'Sec.']: 
+            while line.split() != ['Energy', 'Energy', 'Energy', 'Change', 'Delta', 'Norm', 'in', 'Sec.']:
                 line = next(inputfile)
 
             scfvalues = []
             line = next(inputfile)
             while line.split()[0] != 'Convergence':
                 if line.split()[0].isdigit():
-                    energy = float(line.split()[4].replace('*',''))
-                    density = float(line.split()[5].replace('*',''))
-                    fock = float(line.split()[6].replace('*',''))
-                    dnorm = float(line.split()[7].replace('*',''))
+                    energy = float(line.split()[4].replace('*', ''))
+                    density = float(line.split()[5].replace('*', ''))
+                    fock = float(line.split()[6].replace('*', ''))
+                    dnorm = float(line.split()[7].replace('*', ''))
```

Comment:
Maybe this could be a bit cleaner with `info = line.split()`?

Reply:
Alright!